ui/appui-react/src/appui-react/utils/Placement.ts

```diff
@@ -0,0 +1,62 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+type Side = "Left" | "Top" | "Right" | "Bottom";
```

Comment:
Let's keep this lower case to match floating UI (and iTwinUI types).

Reply:
fixed